tomviz/python/GaussianFilterTiltSeries.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name" : "GaussianFilter",
+  "label" : "Gaussian Filter",
```

Comment:
Could anyone explain this to me? When I edit these two fields, the UI often disappear like this
<img width="202" alt="screen shot 2016-10-24 at 10 39 02 am" src="https://cloud.githubusercontent.com/assets/13088588/19650255/91a1f8da-99d6-11e6-9b01-8b9759b252cc.png">
Thanks!